### ISSUE TITLE
[FEAT]: Expose `pick` method

### ISSUE THREAD

**Issue by TheAlexLichter** [codeacme17/1llest-waveform-vue#9]
### Description

Right now, there is no way to set the time of the waveform component programatically (like you can do with play/pause). Exposing a `pick` method would be ideal!

**Comment by codeacme17** [comment]
Yes, you are right. Providing programmatic pick capability is desirable. Thanks for the suggestion.
